src/main/java/org/apache/commons/logging/LogFactory.java

```diff
@@ -188,7 +189,7 @@
      * AccessControllers etc. It's more efficient to compute it once and
      * cache it here.
      */
-    private static final ClassLoader thisClassLoader;
+    private static final WeakReference<ClassLoader> thisClassLoaderRef;
```

Comment:
CI failing due to this line. I believe the project is using Java 6.

Reply:
I think it was the diamond on L1680 - fixed it.

Reply:
Ops, wrong line, sorry. Thanks!